packages/@aws-cdk/aws-iotevents/lib/action.ts

```diff
@@ -0,0 +1,27 @@
+import * as iam from '@aws-cdk/aws-iam';
+import { CfnDetectorModel } from './iotevents.generated';
+
+/**
+ * An abstract action for DetectorModel.
+ */
+export interface IAction {
+  /**
+   * The policies to perform the AWS IoT Events action.
+   */
+  readonly actionPolicies?: iam.PolicyStatement[];
```

Comment:
Can we move this property to `ActionConfig`?

Reply:
I remove this because `ActionBindOptions` bring the action the role.